### PR TITLE
feat : Category Page added

### PR DESCRIPTION
**Pull request by vikrantkode**
# Open for Reviews
## category page added

### CODE REVIEW

File: Pages/category.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+.card {
+    display: flex;
+    flex-direction: column;
+    position:relative;
+    color: var(--hover-text-color);
+    border-top-left-radius: 0.5rem;
+    border-top-right-radius: 0.5rem;
+    border-bottom-left-radius: 0.5rem;
+    border-bottom-right-radius: 0.5rem;
```
Comment (sharmayatendra) [inline Pages/category.css]: use shorthand property

File: Pages/category.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+.image img{
+    width: 100%;
+    border-top-left-radius: 0.5rem;
+    border-top-right-radius: 0.5rem;
+}
+.title {
+    margin-left: 2rem;
+    margin-top: 1rem;
```
Comment (sharmayatendra) [inline Pages/category.css:43]: same as above

**Review by sharmayatendra — APPROVED** [review]
looks good make suggested changes

File: Pages/category.css (hunk 1 above)
Comment (vikrantkode) [reply Pages/category.css]: done

File: Pages/category.css (hunk 2 above)
Comment (vikrantkode) [reply Pages/category.css:43]: done

**Comment by vikrantkode** [comment]
> looks good make suggested changes

have done the suggested changes

**Review by pnchinmay — APPROVED** [review]

**Pull request merged**